docs-source/content/faq/external-clients.md

```diff
@@ -131,6 +168,8 @@ In this example:
 
 - The operator will automatically create a Kubernetes Service named `DOMAIN_UID-cluster-cluster-1` for both the custom and default channel.
 
+- The operator will automatically set the `ListenAddress` on each WebLogic Server for each of its channels.
+
```

Comment:
"Additional steps are required for external clients beyond configuring the custom channel - see [Approaches](#approaches)."  This x-ref is no longer correct. Not sure how you want to change it.

Reply:
Fixed.